Lab1/app/src/main/java/nm/lab1/MainActivity.java

```diff
@@ -0,0 +1,144 @@
+package nm.lab1;
+
+import android.app.AlertDialog;
+import android.content.DialogInterface;
+import android.support.v7.app.ActionBarActivity;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ListView;
+import android.widget.Toast;
+
+import java.util.ArrayList;
+
+
+public class MainActivity extends ActionBarActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        //define parts of interface
+        Button button = (Button) findViewById(R.id.button);
+        final EditText textbox = (EditText) findViewById(R.id.editText);
+        ListView list = (ListView) findViewById(R.id.list);
+
+        final ArrayList<String> arrayList = new ArrayList<String>();
+        final ArrayAdapter<String> listAdapter = new ArrayAdapter<String>(this,
+                android.R.layout.simple_list_item_1, arrayList);
+
+        list.setAdapter(listAdapter);
+
+        //click button listener
+        button.setOnClickListener(new View.OnClickListener() {
```

Comment:
Your code will look cleaner if you make a separate implementation of the OnClickListener similar to how it is done in this file:
https://github.com/OlinMobileProto/Lab1/blob/master/app/src/main/java/com/example/cwallace/grocerylist/clickListeners/DeleteEntryListener.java